WebHostLib/requirements.txt

```diff
@@ -1,11 +1,11 @@
-flask>=3.0.3
-werkzeug>=3.0.6
+flask>=3.1.0
+werkzeug>=3.1.3
```

Comment:
> Request.max_form_memory_size defaults to 500kB instead of unlimited. Non-file form fields over this size will cause a RequestEntityTooLarge error. [#2964](https://github.com/pallets/werkzeug/issues/2964)

Is there a chance we break this in PlayerOptions?